css/style.css

```diff
@@ -1,6 +1,21 @@
+.navbar-inner {
```

Comment:
I'm a bit confused; the PR summary looks good, but the commit message doesn't?

Some of this work was done a long, long time ago, so I can't make comments on individual formatting changes except put up a publicly-viewable dataset on our instance to let you have a play. Would that help?

P.S. It looks like there are a few commits which I would not have asked to merge (yet) in this PR because I've been committing to the same branch used to generate the PR. I will move them elsewhere and trim the PR back to the original request (primarily: submitting queries in the URL arguments is still experimental).